airflow/providers/google/cloud/transfers/gdrive_to_local.py

```diff
@@ -0,0 +1,112 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from io import BytesIO
+from typing import Optional, Sequence, Union
+
+from airflow.models import BaseOperator
+from airflow.providers.google.suite.hooks.drive import GoogleDriveHook
+from airflow.utils.decorators import apply_defaults
+
+
+class GoogleDriveToLocalOperator(BaseOperator):
+    """
+    Writes a Google Drive file into local Storage.
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:GoogleDriveToGCSOperator`
```

Comment:
```suggestion
        :ref:`howto/operator: GoogleDriveToLocalOperator `
```
This will require a file in `docs/apache-airflow-providers-google/operators/suite/` with how-to guid 👌

Reply:
Do you have a guide how to add a guide?)))

Reply:
There are a bunch of examples in `docs/apache-airflow-providers-google/operators`. 

This was a challenge for me as well. It might be worth it to have a doc similar to [the Contribution Quickstart Guide](https://github.com/apache/airflow/blob/master/CONTRIBUTORS_QUICK_START.rst) but specifically for contributing to google components?